docker-compose-3-nodes.yaml

```diff
@@ -49,7 +49,8 @@ services:
         sawset proposal create \
           -k /etc/sawtooth/keys/validator.priv \
           sawtooth.consensus.algorithm=poet2 \
-          sawtooth.poet.block_claim_delay=0 \
+          sawtooth.consensus.poet2.block_claim_delay=0 \
+          sawtooth.consensus.poet2.key_block_claim_limit=65536 \
```

Comment:
Is this setting in settings_view already?

Reply:
yes